### ISSUE TITLE
[A11y][ScreenReader]NVDA is not announcing the warning suggestion before the reading the line under the Paket CLI command.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9172]
Original: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1555446

Repro Steps:
Hit https://www.nuget.org and press Enter key.
Navigate to "Sign in" control and press Enter key 
Navigate to "Sign in with Microsoft" control and press Enter key
Search for a key term (say “json”) from search box.
Click on "Newtonsoft.json" control and press Enter key
6. Navigate to "Paket CLI" control and press Enter key
Verify that the screen reader is announcing warning suggestion under the paket CLI command or not.

Note:
Similar issue can be observed with the Narrator also.

Actual Result:
NVDA is not announcing the warning suggestion before the reading the line under the Paket CLI command.

Expected result:
NVDA should be announce the warning suggestion before the reading the line under the Paket CLI command.

User Impact:
Screen Reader Users may be get confused whether the warning suggestion is not announced by the screen reader.

Notes:
There is likely an HTML element getting ignored. If not, we can maybe fix this by adding an invisible element with the content we want read out.

**Comment by camigthompson** [comment]
@ryuyu The warning icon that is being ignored by the screen readers has attribute `aria-hidden` set to `true`. Is it expected that the icon be accessible to the screen readers?

**Comment by ryuyu** [comment]
That is a great question. The expectation is that if a warning is presented, the content of the warning should be screen reader accessible. If there is another element that has the same content that is already screen reader accessible, then we can probably call that sufficient. If not, we should either expose the icon, or add a second, screen reader accessible text content.

**Comment by camigthompson** [comment]
The text content of the warning section on the page is screen reader accessible, it is only the icon that is being ignored according to the description of the issue.

**Comment by ryuyu** [comment]
Let me look in to this a bit and figure out what the expectation is.

**Comment by ryuyu** [comment]
Ok so I tested this a little bit and looked at the bug. It appears the expectation is that when you tab to the Paket CLI button in the tab panel, it reads out the warning information, not when the warning information itself is tabbed to. We SHOULD be able to achieve this effect by dynamically adding an aria alert role to the warning element when the tab panel is tabbed to. Similar to what is done in sample 3 of https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role .

**Comment by camigthompson** [comment]
Branch merged into dev.

**Comment by camigthompson** [comment]
Verified in PROD.

**Comment by camigthompson** [comment]
ADO bug has been verified in dev. Issue fixed. Awaiting deployment.
